packages/gatsby-source-shopify/.gitignore

```diff
@@ -0,0 +1,8 @@
+/__tests__
+/yarn.lock
+/*.js
+!index.js
+/gatsby-node.js
```

Comment:
Aren't each of these caught by the `/*.js`?

Reply:
You're right, I brought that over from my repo but it's totally unnecessary. Removed!